post_installation_configuration/cluster-tasks.adoc

```diff
@@ -530,6 +530,18 @@ include::modules/creating-infra-machines.adoc[leveloffset=+2]
 
 * See xref:../architecture/control-plane.adoc#architecture-machine-config-pools_control-plane[Node configuration management with machine config pools] for more information on grouping infra machines in a custom pool.
 
+[id="modify-machine-set-heterogeneous-clusters"]
+== Modifying a machine set for heterogeneous clusters 
+
+To deploy a heterogeneous cluster, you must edit the machine set to include an `arm64` bootimage. The following procedures explain how to generate a `arm64` bootimage and edit the machine set for your heterogeneous infrastructure. Heterogeneous clusters are only available on Azure installer-provisioned infrastructures.
```

Comment:
You must edit or add a new machine set that uses an arm64 image and deploys VMs with an ARM64 architecture

Reply:
ah..yes in fact i would take "edit" out as i recommend in the other thread and just recommend users add new machinesets

Reply:
and so the title would be "Adding a machine set ..."